components/kyma-environment-broker/internal/process/provisioning/event_hub_step.go

```diff
@@ -109,17 +114,19 @@ func (p *ProvisionAzureEventHubStep) Run(operation internal.ProvisioningOperatio
 	log.Printf("Persisted Azure Resource Group [%s]", groupName)
 
 	// create EventHubs Namespace
-	eventHubsNamespace := pp.Parameters.Name
-	eventHubNamespace, err := namespaceClient.CreateNamespace(p.EventHub.Context, azureCfg, groupName, eventHubsNamespace, tags)
```

Comment:
@marcobebway can we use the instance_id instead of name ?

Reply:
Done.